roles/kubernetes_setup/files/scheduler-pod.json

```diff
@@ -0,0 +1,39 @@
+{
+  "kind": "Pod",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "kube-scheduler"
+  },
+  "spec": {
+    "hostNetwork": true,
+    "containers": [
+      {
+        "name": "kube-scheduler",
+        "image": "registry.access.redhat.com/rhel7/kubernetes-scheduler",
```

Comment:
Huh, this is doing containerized kube?  That's an interesting choice.

I'm uncertain about this in the long term, I think the end state we want to get to is supporting:
 - In-system RPM (viable via package layering, will also work on non-Atomic-Host)
 - Kube via system containers

But we haven't talked about this a lot yet...I don't object to testing containerized kube either.